src/TextShow/Data/Aeson.hs

```diff
@@ -31,3 +31,8 @@ instance TextShow Value where
         $ fromText "Object (fromList "
         <> showbPrec 11 (KM.toAscList xs)
         <> singleton ')'
+
+instance TextShow v => TextShow (KM.KeyMap v) where
+    showbPrec d xs = showbParen (d > 10)
+        $ fromText "fromList "
+        <> showbPrec 11 (KM.toAscList xs)
```

Comment:
Can you add a `TextShow1 KM.KeyMap` instance as well?

Reply:
I'm not sure how to make this instance. There is no `liftShowbPrec2` defined for `KeyMap` (not yet). And `showbUnaryListWith` doesn't seem to apply (at least i think so)

Reply:
Something like this should suffice:

```hs
import TextShow (TextShow1(..), ...)
import TextShow.Utils (showbUnaryListWith)

instance TextShow v => TextShow (KM.KeyMap v) where
    showbPrec = showbPrec1

instance TextShow1 KM.KeyMap where
    liftShowbPrec sp sl d xs =
      showbUnaryListWith (liftShowbList sp sl) d (KM.toAscList xs)
```